.gitignore

```diff
@@ -148,3 +148,9 @@ packages/lsp/src/_*
 # generated reports
 webpack-bundle-analyzer.html
 examples/**/test-results/
+
+# untitled (default name) files
```

Comment:
Maybe a more general `[uU]ntitled*` could also be used? (although no strong opinion)

Reply:
Yes, it works:

<img width="1582" alt="image" src="https://github.com/user-attachments/assets/b7f46b4e-9ba5-4001-82f2-0dc740ac6d48">

I don't have a strong opinion either. Typically, I prefer to be more specific when defining these patterns (I find them more readable), but your option seems good to me too! Should I update the PR?